config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -1031,6 +1031,28 @@ periodics:
     testgrid-tab-name: label_sync
     testgrid-alert-email: kubernetes-sig-testing-alerts@googlegroups.com, k8s-infra-oncall@google.com
     description: Runs label_sync to synchronize GitHub repo labels with the label config defined in label_sync/labels.yaml.
+- name: test-infra-merge-testgrid-config-canary
+  interval: 30m
+  cluster: test-infra-trusted
+  max_concurrency: 1
+  decorate: true
+  spec:
+    serviceAccountName: testgrid-config-updater
+    containers:
+    - image: gcr.io/k8s-testgrid/config_merger:latest
+      command:
+      - /config_merger
+      args:
```

Comment:
Thoughts on having this follow the updater pattern of:
* Adding a `--wait=30m` flag that causes it to run forever
* Making this a deployment rather than a periodic release that
  - lives in the testgrid repo
  - becomes part of the testgrid release

Reply:
Having this be part of the TestGrid cluster is a good idea. There will still need to be some changes to the Configurator job here.